app/screens/user_profile/user_profile.js

```diff
@@ -66,9 +64,9 @@ export default class UserProfile extends PureComponent {
     };
 
     displaySendMessageOption = () => {
-        const {currentChannel, currentUserId, user} = this.props;
+        const {currentUserId, user} = this.props;
 
-        return currentChannel.name !== getDirectChannelName(currentUserId, user.id);
+        return user.id !== currentUserId;
```

Comment:
0/5 if we want to remove this so you can loop back and send a message to yourself

Reply:
@saturninoabril I leave this up to you, you can either do this or merge the PR

Reply:
Currently this is something we're doing on the webapp. But I agree on removing that flag so it's easier to send message to self.
Maybe it's something for @esethna to confirm.  I can easily update if needed.

Reply:
We should keep it so you can get to the DM (you) channel.